app/views/shared/_nav.html.erb

```diff
@@ -1,37 +1,33 @@
-<nav class="navbar navbar-expand-sm navbar-dark bg-dark">
-  <%= link_to "Oddio", landmarks_path, class: "navbar-brand" %>
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample03" aria-controls="navbarsExample03" aria-expanded="false" aria-label="Toggle navigation">
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <a class="navbar-brand" href="<%= landmarks_path %>">Oddio</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
-  <div class="collapse navbar-collapse" id="navbarsExample03">
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
     <ul class="navbar-nav mr-auto">
       <li class="nav-item active">
         <a class="nav-link" href="<%= landmarks_path %>">Browse <span class="sr-only">(current)</span></a>
       </li>
       <li class="nav-item">
         <a class="nav-link" href="<%= root_path %>">About</a>
       </li>
-      <li class="nav-item">
-        <div class='search-field'>
-          <%= form_tag landmarks_path, method: :get, class: 'form-inline my-2 my-lg-0' do %>
-            <%= text_field_tag :query, 'Enter a location', class: 'form-control mr-sm-2' %>
-            <%= submit_tag 'Search', class: 'btn btn-primary my-2 my-sm-0' %>
-          <% end %>
-        </div>
-      </li>
       <% if current_user %>
         <li class="nav-item">
-          <a class="btn btn-primary my-2 my-sm-0" href="<%= logout_path %>">Log Out</a>
+          <a class="nav-link" href="<%= logout_path %>">Log Out</a>
         </li>
       <% else %>
         <li class="nav-item">
-          <a class="btn btn-primary my-2 my-sm-0" href="<%= login_path %>">Login</a>
+          <a class="nav-link" href="<%= login_path %>">Login</a>
         </li>
         <li class="nav-item">
-          <a class="btn btn-primary my-2 my-sm-0" href="<%= register_path %>">Register</a>
+          <a class="nav-link" href="<%= register_path %>">Register</a>
         </li>
       <% end %>
     </ul>
+    <form class="form-inline my-2 my-lg-0 pull-right" action="<%= landmarks_path %>">
```

Comment:
May want to keep this part though -- your search box looked better than the original.